### PR TITLE
[Themes] Implement chat background image

### PR DESCRIPTION
**Pull request by amsryq**

### CODE REVIEW

**Comment by Vendicated** [comment]
PREPARE FOR THE ANIME THEME OVERLOAD AND OOOOPEN THE FLOODGATES

**Comment by maisymoe** [comment]
> PREPARE FOR THE ANIME THEME OVERLOAD AND OOOOPEN THE FLOODGATES

image

File: src/lib/themes.ts [hunk 1]
```diff
@@ -19,6 +19,24 @@ async function writeTheme(theme: Theme | {}) {
...
+
+    const MessagesWrapperConnected = findByName("MessagesWrapperConnected", false);
+    if (!MessagesWrapperConnected) return;
+
+    return instead("default", MessagesWrapperConnected, (args, orig) => {
+        return React.createElement(ReactNative.ImageBackground, {
+            style: { flex: 1, height: "100%" },
+            source: { uri: currentTheme.url },
+            blurRadius: currentTheme.blur,
+            // @ts-expect-error - "An outer value of 'this' is shadowed by this container" ???
```
Comment (Vendicated) [inline src/lib/themes.ts]: your instead callback is an arrow function which doesn't capture `this` and instead uses the outer `this`

Changing it to a normal function should fix this issue (or just don't use .apply())

**Comment by Storedband18976** [comment]
how can i download

**Comment by Storedband18976** [comment]
> how can i download 

seriously guys idk how to download

**Comment by aeongdesu** [comment]
> > how can i download
> 
> seriously guys idk how to download

@Storedband18976 you will [join discord server](https://discord.gg/n9QQ4XhhJP)

**Comment by Storedband18976** [comment]
> > > how can i download
> > 
> > seriously guys idk how to download
> 
> @Storedband18976 you will [join discord server](https://discord.gg/n9QQ4XhhJP)

im already in the server
@aeongdesu

**Pull request merged**